<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-QM4RREJ3" name="com.qanonTesting.TestNG.ThirdSession" tests="2" failures="1" timestamp="2020-03-21T10:54:28 EDT" time="414.915" errors="0">
  <testcase name="firstSession" time="207.531" classname="com.qanonTesting.TestNG.ThirdSession">
    <skipped/>
  </testcase> <!-- firstSession -->
  <testcase name="firstSession" time="207.384" classname="com.qanonTesting.TestNG.ThirdSession">
    <failure type="java.lang.AssertionError" message="&#039;FEEDS&#039; PAGE NOT FOUND OR PAGE NAME NOT MATCHED!!!">
      <![CDATA[java.lang.AssertionError: 'FEEDS' PAGE NOT FOUND OR PAGE NAME NOT MATCHED!!!
at org.testng.Assert.fail(Assert.java:97)
at com.qanonTesting.adminMenu.Feeds2.navigateTo_FeedsPage(Feeds2.java:125)
at com.qanonTesting.TestNG.ThirdSession.firstSession(ThirdSession.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- firstSession -->
</testsuite> <!-- com.qanonTesting.TestNG.ThirdSession -->
